NEWS.txt

```diff
@@ -150,6 +150,10 @@ Upgrading
      to TimeWindowCompactionStrategy before upgrading to this version.
    - The deprecated functions `dateOf` and `unixTimestampOf` have been removed. They were deprecated and replaced by
      `toTimestamp` and `toUnixTimestamp` in Cassandra 2.2.
+   - Keystore password configuration is nullable now and the code defaults of the keystore passwords to 'cassandra' are
```

Comment:
@maulin-vasavada we should mention keystore as well as truststore passwords here. It would be also nice to mention what section these configuration parameters are found in.

Reply:
done